packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -126,12 +143,17 @@ export function SchemaDrivenAnnotationEditor(
 
   const [showConfirmation, setShowConfirmation] = React.useState(false)
 
-  const { entityMetadata: entityJson, annotations } = useGetJson(entityId!, {
-    // Metadata is being edited, so don't refetch
-    staleTime: Infinity,
-    enabled: !!entityId,
-    useErrorBoundary: true,
-  })
+  const { entityMetadata: entityJson, annotations } = useGetJson(
```

Comment:
Is it possible that the useGetJson.entityMetadata result would be cached by react-query?

Reply:
Yes. In fact the `entityMetadata` and `annotations` objects are both parts of the `data` that react-query has cached. `useGetJson` splits them for convenience.

The cache entries with/without derived annotations are independent, and both are invalidated when an entity update is made.